script/tasks/diamondMakeImmutable.sh

```diff
@@ -17,22 +17,21 @@ diamondMakeImmutable() {
   checkRequiredVariablesInDotEnv $NETWORK
 
   # define path of JSON file to get diamond address from
-	ADDRS="deployments/$NETWORK.${FILE_SUFFIX}json"
+  ADDRS="deployments/$NETWORK.${FILE_SUFFIX}json"
 
   # get diamond address from path (finds any key that contains "LiFiDiamondImmutable"
-	DIAMOND_ADDRESS=$(jq -r '.LiFiDiamondImmutable' "$ADDRS")
+  DIAMOND_ADDRESS=$(jq -r '.LiFiDiamondImmutable' "$ADDRS")
 
-	# check if all required env variables have values
-	if [[ -z "$PRIVATE_KEY_REFUND_WALLET" || -z "$PRIVATE_KEY_WITHDRAW_WALLET" ]]; then
-	  error "your .env file is missing either PRIVATE_KEY_REFUND_WALLET or PRIVATE_KEY_WITHDRAW_WALLET values. Script cannot continue."
-	  exit 1
+  # check if all required env variables have values
+  if [[ -z "$PRIVATE_KEY_REFUND_WALLET" || -z "$PRIVATE_KEY_WITHDRAW_WALLET" ]]; then
```

Comment:
I think this needs to be updated, too, in order to work with the new solution, since those values also will just contain the name under which the privateKey is stored in the key store. 

I think a helper function getPrivateKey() could still helpful for such cases.